app/main.py

```diff
@@ -16,4 +16,9 @@
 }
 collection_of_coins = {1, 2, 25}
 
-# write your code here
+
+sorted_variables = {
+    "mutable": [my_favourite_films, marks, collection_of_coins],
```

Comment:
The `collection_of_coins` variable is a set, which is an immutable type in Python. It should be moved from the 'mutable' list to the 'immutable' list in the `sorted_variables` dictionary.